.huskyrc

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
```

Comment:
Could you move this config inside package.json? I think it's easier to have all the config there instead of having it in multiple dot files